guides/Building Your Bot/CreatingACustomMemberGateway.md

```diff
@@ -0,0 +1,171 @@
+Something that is often asked or requested, is a custom member-based gateway; that is per member configs, so you would have different {@link Configuration} values for different guilds.
+
+This is simple to achieve with Klasa's advanced Gateway system. In this example, we will be make a per member gateway with a built in key for points. We will be remaking the {@tutorial CreatingPointsSystems} example, in this tutorial.
+
+> Note: I would not recommend using the default (JSON) provider for this, as you may end up reaching the limit of JSON files your OS can open.
+
+## Creating The Gateway
+
+Inside your entry point (Where you define `new Client()`, for example, `index.js`) add this:
+
+```javascript
+client.gateways.register('members', {
+	experience: {
+		type: 'Integer',
+		default: 10
+	},
+	level: {
+		type: 'Integer',
+		default: 1
+	}
+}, { provider: 'rethinkdb' });

```

Comment:
This assumes you have the provider installed. Just don't add this object, and it will use teh default provider.